Makefile

```diff
@@ -1,16 +1,17 @@
-all: build test sanity
-
-.PHONY: build
-build:
-	cargo build --release
+.PHONY: linux
+linux:
+	cargo build --release --target x86_64-unknown-linux-gnu
 	cargo install --path .
 	cargo clippy
 	cargo fmt
-
-.PHONY: test
-test:
 	cargo test --release
-
-.PHONY: sanity
```

Comment:
Also we should not remove this because now we cant run the makefile locally so easily. Maybe we keep a build and test target that we can run locally that does the `cargo fmt, cargo clippy, cargo test --release, cargo install --path ., python3 sanity_check.py`?

Reply:
got it, so separating the makefile into build and test sections are important for running it locally, I'll change it accordanly